ecr.tf

```diff
@@ -0,0 +1,50 @@
+module "ecr_label" {
+  source  = "justtrackio/label/null"
+  version = "0.26.0"
+
+  label_order = var.label_orders.ecr
+  delimiter   = "/"
+
+  context = module.this.context
+}
+
+module "ecr" {
+  source  = "terraform-aws-modules/ecr/aws"
+  version = "1.6.0"
+
+  repository_name = module.ecr_label.id
+  tags            = module.ecr_label.tags
+
+  attach_repository_policy        = false
+  repository_image_scan_on_push   = false
+  repository_image_tag_mutability = "MUTABLE"
+  repository_force_delete         = true
+  repository_lifecycle_policy = jsonencode({
+    rules = [
```

Comment:
where is the rule that keeps all images tagged with the ci default branch as a prefix?

Reply:
there is no reason for it because we will not have any other tags anymore as we don't share the ecr over envs